src/Calculator/SettingsPage.xaml

```diff
@@ -0,0 +1,164 @@
+<Page x:Class="CalculatorApp.SettingsPage"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:Windows10version1809="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 7)"
```

Comment:
This should be renamed to "xmlns:contract7Present" to match the other instances of the Contract 7 API check.